### PR TITLE
Dev/admin

### PR DESCRIPTION
**Pull request by dream184**
1. remove isFollowed attrubute from getFollowings function's return data 
2. debug: when admin deletes tweet, related like data and reply data would be deleted
3. feat: add followsCount and followingsCount to GET api/admin/users
4. feat: user data would be order by tweetsCount in GET api/admin/users
5. debug: filter admin user from all users' data
6. adjust: delete unrequired data in GET api/admin/tweets

### CODE REVIEW

File: services/admin-services.js [hunk 1]
```diff
@@ -24,21 +37,28 @@ const adminController = {
...
           Reply,
+          Tweet,
           { model: User, as: 'Followers' },
           { model: User, as: 'Followings' }
         ],
         group: ['User.id'],
         raw: true,
-        nest: true
+        nest: true,
+        order: Sequelize.literal('max(Tweets.id) DESC')
```
Comment (asakura4) [inline services/admin-services.js:58]: 這個我不太確定跑起來會怎麼樣，晚點抓到local測試

File: services/admin-services.js [hunk 2]
```diff
@@ -1,6 +1,19 @@
 const jwt = require('jsonwebtoken')
-const sequelize = require('sequelize')
+const Sequelize = require('sequelize')
```
Comment (asakura4) [inline services/admin-services.js:2]: [Index.js](https://github.com/dream184/twitter-api-2020/blob/master/models/index.js)裡面已經有設定Sequelize了，
或許可以 
`const  {db} = require('../models/index')`
後直接用他設定好的db.Sequelize，怕這邊多設定環境變數會有問題

**Review by asakura4 — APPROVED** [review]
你那邊跑的過應該就沒問題，先上到Heroku試看看了！！！

**Review by asakura4 — APPROVED** [review]

File: services/admin-services.js (hunk 2 above)
Comment (dream184) [reply services/admin-services.js:2]: config裏頭的production環境變數格式和development格式不同，它是heroku的專屬格式，沒法直接帶入會報錯

**Pull request merged**